tests/framework/extensions/clusters/clusters.go

```diff
@@ -228,6 +233,8 @@ func CreateRKE1Cluster(client *rancher.Client, rke1Cluster *management.Cluster)
 				return false, fmt.Errorf("there was an error deleting cluster")
 			} else if event.Type == watch.Deleted {
 				return true, nil
+			} else if cluster == nil {
```

Comment:
What cluster are you referencing here? When would cluster be nil? If cluster is nil then it would return an error at line 218

Reply:
Should be the cluster the testSession generates. Honestly, I added this check here cause `CreateRKE2Cluster` has the exact same check. I noted that when working on this PR, so added same check for parity.

Reply:
Got it, line has been removed 👍🏿

Reply:
If it is causing issues, just do the same thing, but with a v3.cluster and check if it's nil